src/views/Layout.jsx

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Since the links are now working, we could probably remove this eslint disable line 💥🙏🏻